app/query_param.go

```diff
@@ -8,10 +8,11 @@ import (
 	"github.com/labstack/echo"
 )
 
-func ParseHistoryQueryParams(c echo.Context, defaultLimit int64) (string, int64, int64) {
+func ParseHistoryQueryParams(c echo.Context, defaultLimit int64) (string, int64, int64, bool) {
 	userID := c.QueryParam("userid")
 	from, _ := strconv.ParseInt(c.QueryParam("from"), 10, 64)
```

Comment:
What happens if the parse fails? You have to handle the errors.

Reply:
[Done](https://github.com/topfreegames/mqtt-history/pull/20/commits/1785127750948237ff9da0ccc299116a8b9f29de) 👍🏾